app/controllers/Contributions.scala

```diff
@@ -106,6 +108,8 @@ class Contributions(paymentServices: PaymentServices, addToken: CSRFAddToken) ex
       .map(mobileRedirectUrl)
       .filter(_ => request.isIos)
 
+    info(s"thank you page displayed for paypal payment id: ${request.id}")
+    putThankYouPage()
```

Comment:
I know it's subjective, but I think the brackets are helpful to indicate that evaluating the expression has a side-effect.